sopel/plugins/rules.py

```diff
@@ -343,6 +401,7 @@ class AbstractRule(object):
     def from_callable(cls, settings, handler):
         """Instantiate a rule object from ``settings`` and ``handler``.
 
+        :classmethod:
```

Comment:
Doesn't Sphinx figure this out by itself? (Ditto below, for `kwargs_from_callable`.)

Searching through my (slightly outdated) local HTML docs from the last time I worked on them, there's a _classmethod_ annotation before all instances of these methods in the docs, even inherited ones.

Reply:
It does, nice catch.